frameworks/hdfs/universe/config.json

```diff
@@ -1490,7 +1490,7 @@
         "client_read_shortcircuit": {
           "type": "boolean",
           "description": "This configuration parameter turns on short-circuit local reads.",
-          "default": true
+          "default": false
```

Comment:
We should provide an explanation why we're changing it to false as this may result in degraded performance. Original purpose was to fix the issue in our tests - by changing this we're actually affecting the standard deployment of any HDFS service

Reply:
@samvantran If any process would use HDFS's short circuit read, that process should access `dn_socket` in the HDFS datanode's  mesos sandbox directory. I don't think this socket can be accessed other mesos tasks like spark executors. It is already inaccessible, so it's safe to turn this off. Am I something wrong?